sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -544,12 +493,61 @@ public static JsonSerializable instantiateJsonSerializable(ObjectNode objectNode
                 || klassType.equals(DatabaseAccount.class) || klassType.equals(DatabaseAccountLocation.class)
                 || klassType.equals(ReplicationPolicy.class) || klassType.equals(ConsistencyPolicy.class)
                 || klassType.equals(DocumentCollection.class) || klassType.equals(Database.class)) {
-                return (JsonSerializable) klassType.getDeclaredConstructor(ObjectNode.class).newInstance(objectNode);
+                Constructor declaredConstructor =
+                    klassType.getDeclaredConstructor(ObjectNode.class);
+                declaredConstructor.setAccessible(true);
+                return (JsonSerializable) declaredConstructor.newInstance(objectNode);
             } else {
-                return (JsonSerializable) klassType.getDeclaredConstructor(String.class).newInstance(Utils.toJson(Utils.getSimpleObjectMapper(), objectNode));
+                Constructor declaredConstructor = klassType.getDeclaredConstructor(String.class);
+                declaredConstructor.setAccessible(true);
```

Comment:
same here. if instantiateJsonSerializable is moved to model package, setAccessable should be required.

Reply:
moved it to ModelBridgeInternal.java